tests/rgw/java_s3tests.py

```diff
@@ -51,14 +51,15 @@ def run(**kw):
     host = config.get("host")
 
     client_node = cluster.get_nodes(role="client")[0]
-
-    execute_setup(cluster, config)
-
     if not host:
-        _, secure, _ = get_rgw_frontend(cluster)
+        _, secure, port = get_rgw_frontend(cluster)
 
-    exit_status = execute_s3_tests(client_node, config)
-    execute_teardown(cluster)
+    if "maven" in config["suite"]:
+        exit_status = maven_setup(cluster, config, port)
+    elif "JavaS3" in config["suite"]:
```

Comment:
@TejasC88 , can we use if statement here as well.

Reply:
Done